src/core/Encoding/Registry.pm6

```diff
@@ -21,7 +21,8 @@ my class Encoding::Registry {
             'latin1','latin-1','csisolatin1','l1','ibm819','cp819'),
           nqp::list('windows-1251', 'windows1251'),
           nqp::list('windows-1252', 'windows1252'),
-          nqp::list('windows-932',  'windows932')
+          nqp::list('windows-932',  'windows932'),
+          nqp::list('gb2312',  'gb2312')
```

Comment:
Are trailing commas supported here? If yes, then I'd recommend to write it as `nqp::list('gb2312',  'gb2312'),` so that the next PR for another encoding doesn't have to touch that line.

Reply:
Yes, it's changed in the new commit.